habitat/plan.sh

```diff
@@ -1,6 +1,6 @@
 pkg_name=dcob
 pkg_origin=core
-pkg_version=_computed_
+pkg_version=_computed_in_a_function_below_
```

Comment:
Does this and the comment in `determine_version()` make it more clear that this is a placeholder, not magic?